x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_actions/route.ts

```diff
@@ -112,6 +117,14 @@ const normalizeErrorResponse = (errors: BulkActionError[]): NormalizedRuleError[
   return Array.from(errorsMap, ([_, normalizedError]) => normalizedError);
 };
 
+function isBulkEditError(error: BulkActionError): error is BulkEditError {
+  return (error as BulkEditError).message !== undefined;
+}
+
+export enum BulkEditSkipReason {
+  DataViewExistsAndNotOverriden = 'DataViewExistsAndNotOverriden',
+}
```

Comment:
Unsure where to place this `enum`. Might also be used in the frontend later if we want to explain the user why certain rules where skipped (out of the scope of this PR though)

Reply:
It may make sense to put it apart in a separate file.

Reply:
I'd prefer leaving it here for now. It's used in this endpoint only, and I wouldn't like to create a `constants.ts` file in this dire, or placing it in `/common`, which takes it too far away from its actual usage

Reply:
It should be placed under `security_solution/common/detection_engine/rule_management/api/rules/bulk_actions` folder. Let's create a file called `response_schema.ts` there and move the response interface definition there, including this new enum. I suggested this in a separate comment.